sentry-sample/src/main/AndroidManifest.xml

```diff
@@ -22,6 +22,10 @@
 
     <meta-data android:name="io.sentry.dsn" android:value="https://key@sentry.io/123" />
     <meta-data android:name="io.sentry.debug" android:value="true" />
+
 <!--    <meta-data android:name="io.sentry.auto-init" android:value="false" /> // how to disable the auto-init-->
+
+<!--    <meta-data android:name="io.sentry.ndk" android:value="false" /> // how to disable the NDK-->
+<!--               tools:node="remove" />-->
```

Comment:
@bruno-garcia here is what it needs to be done to disable NDK on the client's side.

Reply:
Does it require the `tools:node="remove" `?
If so, I'm afraid this could become a source of support tickets.

Reply:
yes, if you try to compile it will tell you to do that as a library already did set it up, it's a common problem so developers might know how to solve by eye, but I'd keep this in the documentation.